Dockerfile

```diff
@@ -1,5 +1,44 @@
 ###################
-# STAGE 1: builder
+# STAGE 1.1: builder frontend
+###################
+
+FROM node:15.8.0-alpine as frontend
```

Comment:
When installing Node.js/Yarn inside adoptopenjdk/openjdk8:alpine, we seem to get Node.js v12.20.1. To avoid gotchas and making it as close as possible to the previous Dockerfile, is it possible to use v12.20.1 here instead of v15.8.0?